aws-cli-create-instance.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
Mas, mungkin bisa ditambah step"nya sebelum ke EC2, bagaimana bikin KeyPair, SG dan juga VPC

Reply:
mas @defa-ekky1073 , langkah - langkah saya sebelum bisa utk create Keypair, SG & VPC adalah dengan menambahkan user baru yang punya autentikasi iam tersendiri dan punya role access bertipe AmazonEC2FullAccess. Setelah itu baru saya create keypair baru yang bisa dilihat di line ini :

[aws-cli-create-instance.sh#L3](https://github.com/mario-bros/WHAT-IS-AWS/blob/pertemuan_15/aws-cli-create-instance.sh#L3)

informasi list SG & VPC sudah ada sebelum sy membuatkan user iam baru tersebut, jadi saya tinggal pakai ID SG & VPC yg sudah available mas tanpa sy harus trigger melalui AWS cli